CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing guidelines
```

Comment:
These files are from another PR. Could you remove all the changes that are not part of this PR?

Reply:
I have rebased over `devel`. Probably I should have sent the PR there.